### DISCUSSION TITLE
have remote download magnet function？

### DISCUSSION THREAD

**Discussion by tidexe** [tobychui/arozos#150] [General]
![屏幕截图 2024-02-07 164157](https://github.com/tobychui/arozos/assets/46107860/c7f832e4-adc3-4d77-be2e-de2f3fad4a9d)
just http？

**Comment by tobychui** [comment]
@tidexe also https, but no magnet or bt.
If you want to write one, I can guide you through how to develop a WebApp for arozos. Feel free to contact me for more details :)
